status-server/src/public/js/client.js

```diff
@@ -42,15 +42,77 @@ function init() {
     }
   }
   // populate the LEDs with some initial colors
-  var colors = ['#ffff00', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
+  var colors = ['#ff00ff', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
   colors.forEach((color, idx) => {
     var node = document.getElementById("led" + idx);
     var ctx = node.getContext("2d");
     ctx.fillStyle = color;
     ctx.fillRect(0,0,300,300);
+    ctx.font = "10px Arial";
+    ctx.strokeText(idx ,node.width/2,node.height/2);
+
+
   });
 }
+function animate(ct){
+  var fetchResponse = window.fetch('http://localhost:3000/status'+ct+'.json'+Date.now()).then(
+      function onSuccess(response){
+        return response.json();
+      },
+      function onError(err){
+        console.console.warn("Error fetching"+ url, err);
+      }
+    );
+           updateValueStatus(ct);
+
+}
+
+function updateValueStatus(ct){
```

Comment:
I'm not clear why this function needs to exist - it seems to overlap with updateStatus? Perhaps we should refactor that so we have a updateStatuses which does the forEach loop, and updateStatus updates the individual statusData objects